app/src/main/java/com/dsu/poc/project/ui/login/LoginActivity.kt

```diff
@@ -0,0 +1,4 @@
+package com.dsu.poc.project.ui.login
```

Comment:
This has to be a fragment as discussed. Please maintain a Parent activity with a reasonable name.

Reply:
changed.